legend-engine-xt-flatdata-pure/src/main/resources/core_external_format_flatdata/executionPlan/tests/utils.pure

```diff
@@ -17,9 +17,9 @@ import meta::legend::*;
 import meta::pure::extension::*;
 import meta::pure::runtime::*;
 
-function meta::external::format::flatdata::executionPlan::test::conertStringToByteStream(data:String[1]): InstanceValue[1]
+function meta::external::format::flatdata::executionPlan::test::convertStringToByteArray(data:String[1]): InstanceValue[1]
 {
-  ^InstanceValue(genericType=^GenericType(rawType=ByteStream), multiplicity=PureOne, values=$data)
+  ^InstanceValue(genericType=^GenericType(rawType=Byte), multiplicity=ZeroMany, values=$data)

```

Comment:
Don't we actually need to convert the data here?

Reply:
Do we have a functionality to get bytes from a string in pure?